content/getting-started/yaml.md

```diff
@@ -290,17 +290,40 @@ There are several things to keep in mind about patterns:
 
 ### Publishing
 
-This is the section where you can set up publishing to external services. Codemagic has a number of integrations (e.g. email, Slack, Google Play, App Store Connect) for publishing but you can also use custom scripts to publish elsewhere (e.g. Firebase App Distribution). See the examples [here](../publishing-yaml/distribution).
+Codemagic has a number of integrations (e.g. email, Slack, Google Play, App Store Connect) for publishing but you can also publish elsewhere with custom scripts (e.g. Firebase App Distribution). See the examples [here](../publishing-yaml/distribution).
+
+Note that unlike the scripts the scripts section, the scripts in publishing scripts section are run regardless of the build status. You can specify conditions for running the scripts. For example, you can check whether artifacts were generated or what the build status was.
 
 ```yaml
 publishing:
   email:
     recipients:
       - name@example.com
+  scripts:
+    script: |
+      if ls /path/to/artifacts/**/*.ipa 1> /dev/null 2>&1; then
+        # publish artifacts
+      fi 
+```
+
```

Comment:
This requires some explanation here, e.g.:
You can also use the post-publishing scripts to report build status.